app/src/main/java/org/mozilla/focus/state/AppAction.kt

```diff
@@ -80,4 +80,14 @@ sealed class AppAction : Action {
      * The list of [TopSite] has changed.
      */
     data class TopSitesChange(val topSites: List<TopSite>) : AppAction()
+
+    /**
+     * An item has been added to the list of [TopSite]s.
+     */
+    data class TopSiteAdded(val topSiteUrl: String, val topSiteName: String) : AppAction()
+
+    /**
+     * An item has been removed from the list of [TopSite]s.
+     */
+    data class TopSiteRemoved(val topSiteUrl: String) : AppAction()
```

Comment:
I am very curious if we need these 2 actions since `TopSitesChange` should update the state through its view when topSitesUseCases is called. I will test this out when we have the AC part landed.

Reply:
We need them because the app state should be updated even if we do not have the home screen displayed. The way it is implemented now when you are on the browser screen the app state does not update.

Reply:
Also, making the composition of a composable responsible for updating a state does not seem like a good pattern to me.